apps/finance/app/src/components/Transfers.js

```diff
@@ -29,236 +30,220 @@ const TRANSFER_TYPES = [
 ]
 const TRANSFERS_PER_PAGE = 10
 const TRANSFER_TYPES_STRING = TRANSFER_TYPES.map(TransferTypes.convertToString)
-
 const formatDate = date => format(date, 'dd/MM/yy')
-
 const reduceTokenDetails = (details, { address, decimals, symbol }) => {
   details[toChecksumAddress(address)] = {
     decimals,
     symbol,
   }
   return details
 }
+// Filter transfer based on the selected filters
+const getFilteredTransfers = ({
+  tokens,
+  transactions,
```

Comment:
Since we're moving this out, perhaps we can also modify its arguments.

What do you think about this signature instead?

```js
Transaction = { ... } // current Transaction object
Token = { ... } // current Token object
Range = {
  start: Date,
  end: Date,
}

const getFilteredTransfers = ({
  transactions: Array<Transaction>,
  selectedToken: Token || null,
  selectedTransferType: oneOf(TRANSFER_TYPES),
  selectedDateRange: Range
})
```

Reply:
Done 😄